SoftLayer/CLI/file/order.py

```diff
@@ -51,30 +49,30 @@
 @environment.pass_env
 def cli(env, storage_type, size, iops, tier,
         location, snapshot_size, service_offering, billing):
-    """Order a file storage volume."""
+    """Order a file storage volume.
+    
+    Valid size and iops options can be found here: 
+    https://console.bluemix.net/docs/infrastructure/FileStorage/index.html#provisioning
+    """
     file_manager = SoftLayer.FileStorageManager(env.client)
     storage_type = storage_type.lower()
 
     hourly_billing_flag = False
     if billing.lower() == "hourly":
         hourly_billing_flag = True
 
-    if hourly_billing_flag and service_offering != 'storage_as_a_service':
-        raise exceptions.CLIAbort(
-            'Hourly billing is only available for the storage_as_a_service '
-            'service offering'
-        )
+    if service_offering != 'storage_as_a_service':
+        click.secho('{} is a legacy storage offering'.format(service_offering), fg='red')
+        if hourly_billing_flag:
+            raise exceptions.CLIAbort(
+                'Hourly billing is only available for the storage_as_a_service service offering'
+            )
 
     if storage_type == 'performance':
         if iops is None:
             raise exceptions.CLIAbort(
                 'Option --iops required with Performance')
 
-        if iops % 100 != 0:
-            raise exceptions.CLIAbort(
-                'Option --iops must be a multiple of 100'
-            )
-
         if service_offering == 'performance' and snapshot_size is not None:
             raise exceptions.CLIAbort(
```

Comment:
As with my comment on line 74, I don't think this is a major issue, but I noticed the corresponding exception in SoftLayer/CLI/block/order.py (lines 88 and 89) was updated. If possible, it might be nice to update this one as well for consistency.